api/v1beta1/common.go

```diff
@@ -136,6 +136,9 @@ type AnsibleEESpec struct {
 	ExtraMounts []storage.VolMounts `json:"extraMounts,omitempty"`
 	// Env is a list containing the environment variables to pass to the pod
 	Env []corev1.EnvVar `json:"env,omitempty"`
+    // +kubebuilder:validation:Optional
+    // Extra vars to be passed to ansible process during execution.
```

Comment:
the indent here needs to match the other lines. Is your editor messing with tabs and spaces?